src/canmatrix/formats/cmjson.py

```diff
@@ -192,10 +193,9 @@ def load(f, **options):
             newframe = canmatrix.Frame(frame["name"],
                                    arbitration_id=frame["id"],
                              size=8)
-            if "length" in frame:
-                newframe.size = frame["length"]
+            newframe.size = frame.get("length",0)
```

Comment:
wait, really zero? Or rather the `size=8` from above? I think this code was OK, no need to change.